examples/ui/ui.rs

```diff
@@ -68,7 +72,8 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                                     margin: UiRect::all(Val::Px(5.0)),
                                     ..default()
                                 }),
-                            );
+                                Label,
```

Comment:
Curious why this shouldn't just be included in `TextBundle` as a whole? Alternatively, if that's not what  we want, we could also make a distinct `LabelBundle` for this explicit purpose.

Reply:
We need to distinguish between text that is a label on a button, and text that is a distinct and separate label.

It's possible there should be a `Labelbundle`, but I wanted to be less invasive in how I modified bevy_ui since that's under heavy development, and I think it might make sense to land basic AccessKit integration before doing heavier work on the UI.

For instance, maybe we ditch the `Label` marker, give every distinct widget a `Widget` marker, then labels are just `Text` without other widget type markers.

If you'd like a `Labelbundle` I'm willing to add that, but I'd like more buy-in.

Reply:
I think we can avoid that for now. However, as an example is made to educate, a comment here is probably best to explain why this is necessary.